CONTRIBUTING.md

```diff
@@ -1,23 +1,27 @@
 # Contributing
 
 - [Contributing](#contributing)
-    - [Architecture Decision Records (ADR)](#architecture-decision-records-adr)
-    - [Pull Requests](#pull-requests)
-        - [Pull Request Templates](#pull-request-templates)
-        - [Requesting Reviews](#requesting-reviews)
-        - [Reviewing Pull Requests](#reviewing-pull-requests)
-        - [Updating Documentation](#updating-documentation)
-    - [Forking](#forking)
-    - [Dependencies](#dependencies)
-    - [Protobuf](#protobuf)
-    - [Testing](#testing)
-    - [Branching Model and Release](#branching-model-and-release)
-        - [PR Targeting](#pr-targeting)
-        - [Development Procedure](#development-procedure)
-        - [Pull Merge Procedure](#pull-merge-procedure)
-        - [Release Procedure](#release-procedure)
-        - [Point Release Procedure](#point-release-procedure)
-    - [Code Owner Membership](#code-owner-membership)
+  - [Architecture Decision Records (ADR)](#architecture-decision-records-adr)
+  - [Pull Requests](#pull-requests)
+    - [Pull Request Templates](#pull-request-templates)
+    - [Requesting Reviews](#requesting-reviews)
+    - [Reviewing Pull Requests](#reviewing-pull-requests)
+    - [Updating Documentation](#updating-documentation)
+  - [Forking](#forking)
+  - [Dependencies](#dependencies)
+  - [Protobuf](#protobuf)
+  - [Testing](#testing)
+  - [Branching Model and Release](#branching-model-and-release)
+    - [PR Targeting](#pr-targeting)
+    - [Development Procedure](#development-procedure)
+    - [Pull Merge Procedure](#pull-merge-procedure)
+    - [Release Procedure](#release-procedure)
+    - [Point Release Procedure](#point-release-procedure)
+  - [Code Owner Membership](#code-owner-membership)
+  - [Concept & Release Approval Process](#concept--release-approval-process)
```

Comment:
I'm not sure where this came from

Reply:
Just `$ git checkout master CONTRIBUTING.md`